client/src/modules/Course/Components/Course.tsx

```diff
@@ -279,52 +310,73 @@ export const Course = () => {
               similarCourses={similarCourses}
               isVisible={screenWidth > 768}
             />
-            {/* <div>
-              
-            </div> */}
           </div>
           <div className={styles.rightPanel}>
-            {/* Reviews Displaying */}
             <div className={styles.reviewscontainer}>
               <div className={styles.bar}>
-                <h2 className={styles.title}>Past Reviews ({courseReviews?.length}) </h2>
+                <h2 className={styles.title}>
+                  Past Reviews (
+                  {selectedProf.current !== 'none'
```

Comment:
It's more intuitive for me to have it as just `visibleCourseReviews.length` because otherwise, you might interpret it as being on a second "page" of reviews or something. Can have others give feedback too

Reply:
oh wait this makes a lot of sense!! my logic behind this was that i wanted a way to make sure that users easily know that when a professor is selected they're not seeing all the reviews (esp b/c the statistics on the left sidebar are based on all reviews). i think just changing the number itself works though!